### PR TITLE
fix: safely fetch fields when syncing standard objects

### PR DESCRIPTION
**Pull request by asdfryan**
Tested locally

### CODE REVIEW

File: apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigDetailsPanel.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ function SyncConfigDetailsPanelImpl({ syncConfigId }: SyncConfigDetailsPanelImpl
             periodMs: syncPeriodSecs ? syncPeriodSecs * 1000 : ONE_HOUR_SECONDS,
             strategy,
           },
-          commonObjects: commonObjects.map((object) => ({ object, fetchAllFieldsIntoRaw: true } as CommonObjectConfig)),
+          commonObjects: commonObjects.map((object) => ({ object } as CommonObjectConfig)),
```
Comment (asdfryan) [inline apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigDetailsPanel.tsx:107]: Unrelated change. Just noticed that we are still setting this (it's deleted)

**Review by albertyfwu — APPROVED** [review]
